scripts/python/services/GithubService.py

```diff
@@ -25,17 +26,19 @@ def decorator(*args, **kwargs):
             - Deleting data
             - Updating data
         """
-        github_client_core_api = args[0].github_client_core_api
         try:
             return func(*args, **kwargs)
-        except RateLimitExceededException:
-            logging.warning(
-                "Caught RateLimitExceededException, retrying calls when rate limit resets.")
-            core_api_reset_timestamp = timegm(
-                github_client_core_api.get_rate_limit().core.reset.timetuple())
+        except (RateLimitExceededException, TransportQueryError) as exception:
+            logging.warning(f"Caught {type(exception).__name__}, retrying calls when rate limit resets.")
+            rate_limits = args[0].github_client_core_api.get_rate_limit()
+            rate_limit_to_use = rate_limits.core if type(
```

Comment:
a bit rudimentary, and will probably need to be refactored if we integrate the `search` API as well

Cross that bridge when we come to it though 🌉